test/unit/dirty_associations_test.rb

```diff
@@ -30,4 +75,22 @@ def setup
     @tester.organization_ids = 3
     assert_equal @tester.organization_ids, [3]
   end
+
+  test "should return false if nothing _changed?" do
+    @tester = DummyDirtyAssociationsModel.new
+    refute @tester.organization_ids_changed?
+  end
+
+  test "_changed? should return false after saving" do
+    @first_mock = FirstMock.create(name: "test1")
+    @second_mock = SecondMock.create(first_mock_id: 1)
+    @third_mock = ThirdMock.create(first_mock_id: 1)
+
+    @first_mock.second_mock_ids = []
+    @first_mock.third_mock_ids = []
+    @first_mock.save!
```

Comment:
One request for this test, since it is very much concerned with the state of _changed?, is to assert that _changed? is true before calling save! This defines the behavior we expect more precisely and may help catch regressions in the future.

Reply:
Yes. Thanks for pointing that out. @jturel